### PR TITLE
Updated localhost refs to clickable links

### PR DESCRIPTION
**Pull request by ReubenKogie**
**Comments**
- I didn't change this reference because it related to the example picture for demonstration, hence didn't need to be an active link :-)
Screenshot 2022-05-02 100519

### CODE REVIEW

File: _pages/docs/quick-start-guide/installing.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ It's very easy to get started; just [download the Docker Engine for Linux, Mac,
 
 `docker run -p 5000:80 focisolutions/orbitalmock`
 
-The designer will be running on `http://localhost:4200`, and the server on `http://localhost:5000`.
+The designer will be running on [http://localhost:4200](http://localhost:4200), and the server on [http://localhost:5000](http://localhost:5000).
```
Comment (glitch452) [inline _pages/docs/quick-start-guide/installing.md]: The markdown spec includes a shorthand for turning a url into a link without title text, could do this to make it a little cleaner.
```suggestion
The designer will be running on <http://localhost:4200>, and the server on <http://localhost:5000>.
```

**Review by glitch452 — CHANGES_REQUESTED** [review]

File: _pages/docs/quick-start-guide/installing.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Install [Docker Compose](https://docs.docker.com/compose/install/)
 
 `docker-compose up -d`
 
-The designer will be running on `http://localhost:4200`, and the server on `http://localhost:5000`.
+The designer will be running on [http://localhost:4200](http://localhost:4200), and the server on [http://localhost:5000](http://localhost:5000).
```
Comment (glitch452) [inline _pages/docs/quick-start-guide/installing.md]: Don't forget about here!

File: _pages/docs/quick-start-guide/installing.md [hunk 3]
```diff
@@ -65,7 +64,7 @@ When building from the source, the server will run on port `5001`.
 
 `cd Orbital/src/Orbital.Mock.Server`
 
-`dotnet run` (defaults to `http://localhost:5000`)
+`dotnet run` (defaults to [http://localhost:5000](http://localhost:5000) and [https://localhost:5001](https://localhost:5001))
```
Comment (glitch452) [inline _pages/docs/quick-start-guide/installing.md]: Don't forget about here!

File: _pages/docs/quick-start-guide/installing.md [hunk 4]
```diff
@@ -75,4 +74,4 @@ When building from the source, the server will run on port `5001`.
 
 `npm run build`
 
-`npm run start` (defaults to `http://localhost:4200`)
+`npm run start` (defaults to [http://localhost:4200](http://localhost:4200))
```
Comment (glitch452) [inline _pages/docs/quick-start-guide/installing.md]: Don't forget about here!

**Review by glitch452 — CHANGES_REQUESTED** [review]

File: _pages/docs/quick-start-guide/installing.md (hunk 2 above)
Comment (ReubenKogie) [reply _pages/docs/quick-start-guide/installing.md]: Oh yes, just noticed I'll need to do it in multiple spots - will push updates!

**Comment by ReubenKogie** [comment]
@glitch452 , @maksvet  - pushed those updates through, please review.
For some reason it included the .NET revisions which I made on a separate local sub-branch but propagated through this branch (didn't stash?)

**Review by glitch452 — APPROVED** [review]

**Pull request merged**